recipes/rtmidi/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class RtMidiConan(ConanFile):
+    name = "rtmidi"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.music.mcgill.ca/~gary/rtmidi/"
+    description = "Realtime MIDI input/output"
+    topics = ("midi")
+    license = "Copyright (c) 2003-2019 Gary P. Scavone"
```

Comment:
The license is MIT+send-patches-upstream.

Reply:
Did you find a name reference for this license? I can find more projects using it, but I didn't find a common name. Anyway, send-patches-upstream sounds good.

Reply:
I scanned spdx and found nothing similar.

Reply:
I found more projects using same recipe. But none of them refer a name too. Your suffix is good enough. I'll take a look in the process to make it official.